source/extensions/omni.isaac.lab_tasks/omni/isaac/lab_tasks/manager_based/classic/cartpole/cartpole_env_cfg.py

```diff
@@ -87,6 +87,12 @@ class EventCfg:
     """Configuration for events."""
 
     # reset
+    reset_to_default = EventTerm(
```

Comment:
would this change affect other environments that use reset_joints_by_offset? such as ant, humanoid, cabinet

Reply:
This isn't a place to have this feature as a unit test IMO. We're changing the way the environment operates.